content/javascript/concepts/npm-in-javascript/npm-in-javascript.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: "NPM in JavaScript"
+Subjects:
+  - "Bash/Shell"
+  - "Developer Tools"
+  - "Web Development"
+Tags: 
+  - "Bash/Shell"
+  - "Command Line"
+  - "Node"
+  - "NPM"
+  - "Packages"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-node-js"
+  - "https://www.codecademy.com/learn/paths/full-stack-engineer-career-path"
+---
+
+Node Package Manager (NPM) is a large collection of JavaScript packages. This makes adding complexity and funcionality to JavaScript project seamless. NPM is also the official package manager of Node.js. Therefore, if Node.js was previously downloaded, NPM is already installed! 
+
+## Packages
+
+Packages are tools composed of files and folers of JavaScript code. They provide one or more operations that can be installed, imported and used in a separate project. Thanks to NPM, the installation process for packages is uniform.
+
+
+## The NPM Registry
+
+NPM comes as two primary tools: 
+
+* a command-line tool with abilities that include installing and uninstalling packages.
```

Comment:
The Curriculum team usually capitalize the first word in bullet points if it's a full sentence.